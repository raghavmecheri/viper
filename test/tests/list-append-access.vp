string[] names = ["trey"];
print(names[0]);

names.append("ratghav");
print(names[1]);

int[] nums = [1];
print(nums[0]);

nums.append(2);
print(nums[1]);

char[] chars = ['A'];
print(chars[0]);

chars.append('B');
print(chars[1]);

float[] nums2 = [1.0];
print(nums2[0]);

nums2.append(2.0);
print(nums2[1]);