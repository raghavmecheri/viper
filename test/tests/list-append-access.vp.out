trey
ratghav
1
2
A
B
1
2